.github/workflows/pdf_generate.yml

```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
+
+name: PDF Generation 
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
```

Comment:
Do we want to add a step to publish the pdfs so we can link to them?

Reply:
Yup, that should be in there now.